res/qml/Theme/Theme.qml

```diff
@@ -1,66 +1,69 @@
-import QtQuick 2.12
 pragma Singleton
+import QtQuick 2.12
 
 QtObject {
-    property color white: "#e3d7fb"
-    property color yellow: "#fca001"
-    property color red: "#ea2a4e"
```

Comment:
can' you recall what you actually changed here? It's difficult to tell because the diff is so messed up.

Reply:
Turns out that I had made 5 changes, but then decided to re-order alphabetically all keys to help with searching for a given property.
I've broke it into two commits to help review:
- 9da1956891d9f2f145a38b71394f2fd375321148
- 28788bf4032228f4ff5bd2d9ac4775c16dc4b377